sdc/hiframes/pd_series_ext.py

```diff
@@ -573,10 +573,10 @@ def _resolve_map_func(self, ary, args, kws):
     def resolve_map(self, ary, args, kws):
         return self._resolve_map_func(ary, args, kws)
 
-    @bound_function("series.apply")
-    def resolve_apply(self, ary, args, kws):
-        # TODO: handle apply differences: extra args, np ufuncs etc.
-        return self._resolve_map_func(ary, args, kws)
+    # @bound_function("series.apply")
+    # def resolve_apply(self, ary, args, kws):
+    #     # TODO: handle apply differences: extra args, np ufuncs etc.
+    #     return self._resolve_map_func(ary, args, kws)
```

Comment:
If you add `'resolve_apply'` to the list `_non_hpat_pipeline_attrs` in `pd_series_ext.py`, then you don't need to comment out these lines.

Reply:
Shouldn't we leave `apply` in "old style" for `SDC` pipeline (`SDC_CONFIG_PIPELINESDC=1`)?

Reply:
Eventually I think we can leave the lines without changes.

Reply:
Done. I have used `_non_hpat_pipeline_attrs` to switch off old implementation.